### PR TITLE
Release v2.2.0 preparation

### PR DESCRIPTION
**Pull request by ed-xmos**
https://xmosjira.atlassian.net/browse/AP-196

- [ ]  Remove bad link workaround when fwk_rtos updated
- [x] Update to released version fwk_voice 0.7.0

### CODE REVIEW

File: settings.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 project: sln_voice
 title: XCORE-VOICE SOLUTION
-version: 2.1.0
+version: 2.2.0
```
Comment (mbanth) [inline settings.yml:4]: Are we sure that this is the only place that needs a version number update?  While I would hope that is the case, experience has taught me that many products and reference designs actually hard code the version number in multiple places.

**Review by mbanth — APPROVED** [review]
Everything looks okay.  I have raised one concern, but it is not backed up by any evidence.

File: settings.yml (hunk 1 above)
Comment (ed-xmos) [reply settings.yml:4]: Yes - I recursively searched for 2.1.0 and 2.0.0 and there were no matches outside of changelog and modules.

**Comment by ed-xmos** [comment]
> Everything looks okay. I have raised one concern, but it is not backed up by any evidence.

This is not the final merge - there are a couple more to develop, then release staging. I have asked Tom about what sort of final review we need.

**Pull request merged**